src/main/java/org/opensearch/security/securityconf/ConfigModelV7.java

```diff
@@ -366,19 +348,29 @@ public EvaluatedDlsFlsConfig getDlsFls(User user, IndexNameExpressionResolver re
             Map<String, Set<String>> dlsQueriesByIndex = new HashMap<String, Set<String>>();            
             Map<String, Set<String>> flsFields = new HashMap<String, Set<String>>();
             Map<String, Set<String>> maskedFieldsMap = new HashMap<String, Set<String>>();
-                        
+
+            // we capture all concrete indices that do not have any
+            // DLS/FLS/Masked Fields restrictions. If the dmf_empty_overwrites_all
```

Comment:
nit-picking:
```suggestion
            // DLS/FLS/Masked Fields restrictions. If the dfm_empty_overwrites_all
```

i have to say: this line finally explained to me what "dfm" seems to stand for: DLS/FLS/Masked 😄. maybe it'd make sense to document the meaning of this abbreviation somewhere?

Reply:
please note that this isn't resolved yet even though it has been marked as such.